tests/rbd/rbd_failback.py

```diff
@@ -0,0 +1,85 @@
+import time
+
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Verification of rbd mirroring failback scenarios on primary site,
+    This module configures the mirroring back to failback cluster.
+
+    Args:
+        kw: test data
+
+    Returns:
+        int: The return value - 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-9476- Primary cluster permanent failure,
+    Recreate the primary cluster and Re-establish mirror with newly created cluster as Primary.
+    Pre-requisites :
+    1. Cluster must be up and running with capacity to create pool
+    (At least with 64 pgs)
+    2. We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test case flows:
+    1) Create a pool with same name as secondary pool has
+    1) Perform the mirroring bootstrap for cluster peers
+    2) copy and import bootstrap token to peer cluster
+    3) verify peer cluster got added successfully after failback
+    4) verify all the images from secondary mirrored to primary
+    """
+    try:
+        log.info("Re-establishing mirror with newly created cluster as Primary")
+        config = kw.get("config")
+        mirror1 = rbdmirror.RbdMirror(
+            kw.get("ceph_cluster_dict").get("ceph-rbd1"), config
+        )
```

Comment:
We could use the initial_mirror_config method right?

Reply:
https://github.com/red-hat-storage/cephci/blob/master/tests/rbd_mirror/rbd_mirror_utils.py#L276
`initial_mirror_config` creates pool on both cluster  and establishes mirroring on both but in this scenario it's not needed as pool needs to be create only on primary cluster.
Hence not used that method.